utils/src/main/java/org/robolectric/util/inject/Supersedes.java

```diff
@@ -10,7 +10,7 @@
 @Documented
 @Retention(RetentionPolicy.RUNTIME)
 @Target(ElementType.TYPE)
-public @interface Supercedes {
+public @interface Supersedes {
```

Comment:
If it is public, and user can access it, this change maybe cause ClassNotFound problem directly. Do we need add new class with correct name, and deprecated old one, and delete it next release(maybe 4.8)?

Reply:
Generally, yes, but in this case, I think it's safe to make this change. I honestly don't think there are any external users (e.g. https://github.com/search?q=org.robolectric.util.inject.Supersedes), this is more of an internal API used by Google to experiment with new Robolectric features and with rollouts.

If there end up being external users I'd be happy to roll this back.